README.md

```diff
@@ -17,6 +17,7 @@ This module requires the following modules/libraries:
 * [Islandora](https://github.com/islandora/islandora)
 * [Tuque](https://github.com/islandora/tuque)
 * [Tesseract](https://github.com/tesseract-ocr/tesseract/wiki)
+* [Islandora Solr Search](https://github.com/Islandora/islandora_solr_search)
```

Comment:
I don't think Solr is a dependency.  It should probably be documented something along the lines of `Further functionality will be enabled if  _module_ is enabled.`.

Reply:
According to https://jira.duraspace.org/browse/ISLANDORA-1564 ticket it was suggested me to put this as hard dependency.

Reply:
@willtp87 Ok I wold like to clear this issue also. Waiting for call.